_overviews/scala3-book/concurrency.md

```diff
@@ -55,7 +55,8 @@ You’ll also see examples of methods that are used to handle the value in a fut
 
 > When you think about futures, it’s important to know that they’re intended as a one-shot, “Handle this relatively slow computation on some other thread, and call me back with a result when you’re done” construct.
 > As a point of contrast, [Akka](https://akka.io) actors are intended to run for a long time and respond to many requests during their lifetime.
-> While an actor may live forever, a future is intended to be run only once.
+> While an actor may live forever, a future eventually contains the result
+> of a computation that ran only once.
```

Comment:
I think it is important to not say things like “we run a `Future`” because that maintains a confusion on the purpose of `Future` (which only holds a future result, unlike a “task”, which is a computation that we can run, cancel, etc.)